modules/sr/robot3/camera.py

```diff
@@ -191,20 +192,21 @@ def _see(self) -> list[Marker]:
                 recognition_model,
             )
             if marker_info:
-                object_infos[recognition_object] = marker_info
+                recognition_objects.append(recognition_object)
+                object_infos[recognition_object.getModel()] = marker_info
```

Comment:
`getModel` just returns the contents of the model field which does not have a unique constraint. I think this can be done by using `getId` instead which returns the object's ID in the scene tree.